### DISCUSSION TITLE
TCP sockets keep alive timeout ?

### DISCUSSION THREAD

**Discussion by flashultra** [libuv/libuv#3951] [Q&A] [answered]
Currently,  it's not possible to  change the keep alive timeout for TCP sockets connections.

I found a possible solution here : https://github.com/libuv/help/issues/54#issuecomment-396519438 , and this  workaround works on pretty much all major platforms .

 Since we are using an unmodified instalation  of libuv for production , are there any reasons why this is not included in the libuv ?

Thank you.

**Comment by bnoordhuis** [comment, marked as answer]
`uv_tcp_keepalive(handle, delay)` exists but if that isn't what you mean, then the answer probably is "not cross-platform."
